src/planner/Planner.cpp

```diff
@@ -18,6 +23,12 @@ statespace::ConstStateSpacePtr Planner::getStateSpace() const
   return mStateSpace;
 }
 
+//==============================================================================
+common::RNG* Planner::getRng() const
```

Comment:
This violates const-correctness. It should be `common::RNG* getRng()`. I think const version (i.e., `const common::RNG* getRng() const`) wouldn't make sense because RNG needs to be non-const as the random generation operator isn't const.